EIPS/eip-1523.md

```diff
@@ -0,0 +1,122 @@
+type: Standards Track
+category: ERC
+created: 2018-10-10
+requires: 721
+---
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+A standard interface for insurance policies, based on ERC 721. 
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+The following standard allows for the implementation of a standard API for insurance policies within smart contracts. 
+Insurance policies are financial assets which are unique in some aspects, as they are connected to a customer, a specific risk, or have other unique properties like premium, period, carrier, underwriter etc.
+Nevertheless, there are many potential applications where insurance policies can be traded, transferred or otherwise treated as an asset.
+The ERC 721 standard already provides the standard and technical means to handle policies as a specific class of non fungible tokens.
+insurance In this proposal, we define a minimum metadata structure with properties which are common to the greatest possible class of policies.
+
+## Motivation
+<!--The motivation is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.-->
+For a decentralized insurance protocol, a standard for insurance policies is crucial for interoperability of the involved services and application.
+It allows policies to be bundled, securitized, traded in a uniform and flexible way by many independent actors like syndicates, brokers, and insurance companies.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+An ERC-1523 compliant insurance policy is a non-fungible token which **MUST adhere to the ERC-721 token standard** and **MUST implement the
+ERC721Metadata and the  ERC721Enumerable interface**.
+
+The implementor MAY choose values for the ```name``` and ```symbol```.
+
+The **policy metadata extension** is **RECOMMENDED** for ERC-1523 smart contracts. 
+This allows your smart contract to be interrogated for policy metadata.
+
+```solidity
+/// @title ERC-1523 Insurance Policy Standard, optional policy metadata extension
+/// @dev See ...
+///  Note: the ERC-165 identifier for this interface is 0x.....
+interface ERC1523PolicyMetadata /* is ERC1523 */ {
```

Comment:
Do you mean `is ERC721` here?

Reply:
ERC1523PolicyMetadata inherits ERC1532 which inherits ERC721 which inherits ERC165, so I think /* is ERC1523 */ is correct, or am I wrong?

Reply:
I don't see where `ERC1523` is defined though.

Reply:
I added an explicit definition of the interface. However, the EIP-1523 standard does not add additional functionality, just defines that ERC721Metadata and ERC721Enumberable MUST be implemented and how policyMetadata (which is optional) SHOULD be implemented.

Reply:
Ah OK! It was just confusing reading first. Hopefully when Solidity allows inheritance in interfaces this will get better. Thanks for updating the ERC!

Are you ok merging it now?

Reply:
yes fine for me, thanks for your comments, helped to clarify a lot.